core/lib/slider.spec.ts

```diff
@@ -682,6 +682,101 @@ describe(`Slider basic`, () => {
 
 		expect(state).toStrictEqual(expectedStateValue);
 	});
+
+	test(`should correctly increment the handle value when the min is decimal and step is integer`, () => {
+		slider.patch({
+			min: 0.5,
+			max: 3.5,
+			stepSize: 1,
+			values: [0.5],
+		});
+
+		const expectedStateValue = {...defaultStateValues};
+		expectedStateValue.min = 0.5;
```

Comment:
No need to redefine the values that does not change. 
It's a little bit misleading, as we can't see at a glance what are the impact of an action.

Reply:
these values change with regard of the default values (that have 0 as min, 100 as maximum and [0] as values), so here it is initial assignment of values for this test case